packages/cli/src/lib/cmds/shared.ts

```diff
@@ -84,3 +88,58 @@ export async function updateVersions(
     appVersionName: appVersionName,
   };
 }
+
+export async function generateManifestChecksumFile(manifestFile: string,
+    prompt: Prompt): Promise<void> {
+  fs.readFile(manifestFile, async function(err, data) {
+    if (err) {
+      prompt.printMessage(err.toString());
+      return;
+    }
+    const csFile = path.join(process.cwd(), 'manifest-checksum.txt');
+    const sum = crypto.createHash('sha1').update(data).digest('hex');
```

Comment:
You've got this line in two places - here and in build.ts line 67, and both lines need to stay consistent. Could you create some helper function (`computeChecksum`?) so we don't risk someone accidentally changing one but not the other?

Reply:
Good call! Added the helper function in.